.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: java
+
+jdk:
+  - oraclejdk8
+
+notifications:
+  email: false
```

Comment:
Will be good to have email notifications. We use "process@snappydata.io" for github notifications so that same can be used here.

Reply:
Pull Request builds do not trigger email notifications in Travis. https://docs.travis-ci.com/user/notifications/#Configuring-email-notifications 

Emails are sent when, on the given branch: 
 - a build was just broken or still is broken.
 - a previously broken build was just fixed.
It is possible to change this frequency and the default conditions that trigger the notification.

The process email ID should work since it's already configured in GitHub. 

notifications:
  email:
    recipients: 
      - process@snappydata.io